seed/challenges/01-front-end-development-certification/basic-javascript.json

```diff
@@ -2419,6 +2427,10 @@
         "",
         "// Only change code below this line",
         "",
+        "",
+        "// Only change code above this line",
+        "",
+        "timesFive(5);",
```

Comment:
I haven't received any errors. The reason I put those there, as well as the `// Only change code above this line` comment was to keep a uniform appearance and experience with other challenges that have similar setups. Especially the function call at the bottom: just about every challenge follows that pattern.